docs/website/docs/dlt-ecosystem/verified-sources/airtable.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Airtable
+description: dlt verified source for Airtable
+keywords: [airtable api, airtable verified source, airtable]
+---
+
+# Airtable
+
+Airtable is a cloud-based platform that merges spreadsheet and database functionalities for easy
+data management and collaboration.
+
+This Airtable `dlt` verified source and
+[pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/airtable_pipeline.py)
+loads data using “Airtable API” to the destination of your choice.
+
+Sources and resources that can be loaded using this verified source are:
+
+| Name              | Description                                |
+| ----------------- | ------------------------------------------ |
+| airtable_source   | Retrieves tables from an airtable base     |
+| airtable_resource | Retrives data from a single airtable table |
+
+## Setup Guide
+
+### Grab Airtable personal access tokens
+
+1. Click your account icon top-right.
+1. Choose "Developer Hub" from the dropdown.
+1. Select "Personal access token" on the left, then "Create new token".
+1. Name it appropriately.
+1. Add read scopes for "data records" and "schema bases".
+1. Add a base to specify the desired base for data access.
+1. Hit "Create token" and save the displayed token securely for credential use.
+
+### Grab Airtable IDs
+
+Upon logging into Airtable, you'll notice a URL in your browser's address bar resembling:
+
+```bash
+https://airtable.com/appve10kl227BIT4GV/tblOUnZVLFWbemTP1/viw3qtF76bRQC3wKx/rec9khXgeTotgCQ62?blocks=hide
```

Comment:
this link is public? I have no permission for it..

Reply:
Indeed, my original one was. Anna or Marcin kindly recreated this base in your dlthub Airtable account. But, I forgot to mention that this has to be shared publicly.

I can't do it since I'm not the owner. Can you log into that account and flip the following switch?
<img width="940" alt="Screenshot 2023-09-19 at 17 35 32" src="https://github.com/dlt-hub/dlt/assets/217980/cb5bd823-f88a-4291-91e1-cd1951d851b7">

Reply:
I wouldn’t set our test account to default, you never know :D

Reply:
The URL is from Airtable official docs and match the image we used. I think it's apt.